src/analyses/README.md

```diff
@@ -32,7 +32,10 @@ To be documented.
 
 \subsection analyses-constant-propagation Constant propagation (constant_propagator_ait)
 
-To be documented.
+A simple, unsound constant propagator. Replaces RHS symbol expressions (variable
```

Comment:
There should be a link to the file and class.

Reply:
Added (in heading)